src/app/layout/qgslayoutdesignerdialog.cpp

```diff
@@ -717,8 +719,8 @@ QgsLayoutDesignerDialog::QgsLayoutDesignerDialog( QWidget *parent, Qt::WindowFla
   QValidator *zoomValidator = new QRegularExpressionValidator( zoomRx, mStatusZoomCombo );
   mStatusZoomCombo->lineEdit()->setValidator( zoomValidator );
 
-  const auto constSStatusZoomLevelsList = sStatusZoomLevelsList;
-  for ( double level : constSStatusZoomLevelsList )
+  const auto statusZoomLevelsList = *sStatusZoomLevelsList();
+  for ( double level : statusZoomLevelsList )
```

Comment:
```suggestion
  for ( double level : { 0.125, 0.25, 0.5, 1.0, 2.0, 4.0, 8.0} )
```

no more static variable required!